src/components/AccountMultisigGraph/AccountMultisigGraphTs.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 NEM (https://nem.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ *
+ */
+import { Component, Prop, Vue } from 'vue-property-decorator';
+// internal dependencies
+import { AccountModel } from '@/core/database/entities/AccountModel';
+import { mapGetters } from 'vuex';
+import { Address } from 'symbol-sdk';
+
+@Component({
+    computed: {
+        ...mapGetters({
+            multisigAccountGraphInfo: 'account/multisigAccountGraph',
+            knownAccounts: 'account/knownAccounts',
+        }),
+    },
+})
+export class AccountMultisigGraphTs extends Vue {
+    @Prop({
+        default: null,
+    })
+    account: AccountModel;
+
+    @Prop({
+        default: null,
+    })
+    visible: boolean;
+    public multisigAccountGraphInfo: [];
+    public knownAccounts: AccountModel[];
+
+    get multisigGraphTree(): any[] {
+        if (this.multisigAccountGraphInfo) {
+            const tree = [];
+            this.multisigAccountGraphInfo.map((level: []) => {
+                level.map((entry) => {
+                    // @ts-ignore
+                    const selected = this.account.address === entry.accountAddress.address ? true : false;
```

Comment:
Do we need a ternary operator here? instead => `const selected = this.account.address === entry.accountAddress.address;`